distro/nix/greeter.nix

```diff
@@ -38,7 +38,7 @@ in {
     options.programs.dankMaterialShell.greeter = {
         enable = lib.mkEnableOption "DankMaterialShell greeter";
         compositor.name = lib.mkOption {
-            type = types.enum ["niri" "hyprland" "sway"];
+            type = types.enum ["niri" "hyprland" "sway" "scroll"];
```

Comment:
Modules/Greeter/assets/dms-greeter would also need to be modified, to actually support running the greeter on scroll